app/models/content_item.rb

```diff
@@ -28,6 +28,15 @@ class ContentItem < ActiveRecord::Base
   NON_RENDERABLE_FORMATS = %w(redirect gone).freeze
   EMPTY_BASE_PATH_FORMATS = %w(government).freeze
 
+  has_one :state
+  has_one :location
+  has_one :translation
+  has_one :user_facing_version
+
```

Comment:
There's quite a few places in publishing api that would benefit from eager loading, and the easiest way to do this is to let Rails manage it for us. I think while we have this multi-table model it makes sense to keep access between the objects as convenient as possible. As a new developer on publishing api I'd expect these associations to exist. Calling `Thing.find_by!(content_item: content_item)` in so many places seems wrong, and doesn't allow Rails to memoize objects correctly. This PR is one example, but if the calling code of this had also had its objects eager loaded, there would be just a single query at the end to find duplicates.